quickwit/quickwit-search/src/metrics.rs

```diff
@@ -34,6 +38,36 @@ pub struct SearchMetrics {
 impl Default for SearchMetrics {
     fn default() -> Self {
         SearchMetrics {
+            root_search_requests_total: new_counter_vec(
+                "root_search_requests_total",
+                "Total number of root search gRPC requests processed.",
+                "search",
+                &[("kind", "server")],
+                ["status"],
+            ),
+            root_search_request_duration_seconds: new_histogram_vec(
+                "root_search_request_duration_seconds",
+                "Duration of request in seconds.",
+                "search",
+                &[("kind", "server")],
+                ["status"],
+                exponential_buckets(0.001, 2.0, 12).unwrap(),
+            ),
+            leaf_search_requests_total: new_counter_vec(
+                "leaf_search_requests_total",
+                "Total number of gRPC requests processed.",
+                "search",
+                &[("kind", "server")],
+                ["status"],
+            ),
+            leaf_search_request_duration_seconds: new_histogram_vec(
+                "leaf_search_request_duration_seconds",
+                "Duration of request in seconds.",
+                "search",
+                &[("kind", "server")],
+                ["status"],
+                exponential_buckets(0.001, 2.0, 12).unwrap(),
```

Comment:
line 82 (for leaf_search_split_duration_secs), the parameters are  `(0.005, 2.0, 10)`. We should use the same parameters (no big opinion on which is better, but having 5ms precision on split, and 1ms on leaf search seems the wrong way around)

Reply:
on second though, the number of buckets should probably be higher. https://github.com/quickwit-oss/quickwit/issues/5443 requests bucket up to 30s, so `(0.005, 2.0, 13)` or `(0.001, 2.0, 15)`

Reply:
Thanks for spotting this. I used `(0.001, 2.0, 15)` for both RPCs.